### PR TITLE
New content for dry-run install

### PR DESCRIPTION
**Pull request by mdeneva**
Added some new content to the kubectl page. Please review and suggest whether we need more or something I say is not technically correct, thanks :)

### CODE REVIEW

File: docs/reference/kubectl-plugin.md [hunk 1]
```diff
@@ -98,7 +98,12 @@ Resource declaratively, as a yaml.
     #        - "true"
     ```
 
-1. Install cluster
+   and install a cluster
     ```bash
     kubectl storageos install --stos-cluster-yaml StorageOSCluster.yaml --etcd-endpoints "storageos-etcd-client.storageos-etcd:2379"
     ```
+* Create a YAML to describe the cluster's resources using a [Heml chart](https://github.com/storageos/charts/pull/129) or use the kubectl plugin with the dry-run flags enabled:
```
Comment (cvlc) [inline docs/reference/kubectl-plugin.md:105]: Heml -> Helm

File: docs/reference/kubectl-plugin.md [hunk 2]
```diff
@@ -98,7 +98,12 @@ Resource declaratively, as a yaml.
     #        - "true"
     ```
 
-1. Install cluster
+   and install a cluster
     ```bash
     kubectl storageos install --stos-cluster-yaml StorageOSCluster.yaml --etcd-endpoints "storageos-etcd-client.storageos-etcd:2379"
     ```
+* Create a YAML to describe the cluster's resources using a [Heml chart](https://github.com/storageos/charts/pull/129) or use the kubectl plugin with the dry-run flags enabled: 
+`kubecl storageos install --username c3RvcmFnZW9z --password c3RvcmFnZW9z --include-etcd ... --dry-run`
```
Comment (cvlc) [inline docs/reference/kubectl-plugin.md:106]: kubecl -> kubectl

File: docs/reference/kubectl-plugin.md [hunk 3]
```diff
@@ -98,7 +98,12 @@ Resource declaratively, as a yaml.
     #        - "true"
     ```
 
-1. Install cluster
+   and install a cluster
     ```bash
     kubectl storageos install --stos-cluster-yaml StorageOSCluster.yaml --etcd-endpoints "storageos-etcd-client.storageos-etcd:2379"
```
Comment (Arau) [inline docs/reference/kubectl-plugin.md:103]: This is not part of the change, but we are here, so I'd compact the shell lines for readability

```
kubectl storageos install \
    --stos-cluster-yaml StorageOSCluster.yaml \
    --etcd-endpoints "storageos-etcd-client.storageos-etcd:2379"
```

**Review by Arau — COMMENTED** [review]
I'm happy to approve following the changes that @cvlc mentioned about the cosmetic typos. Happy to merge when those are fixed 😄

**Review by cvlc — CHANGES_REQUESTED** [review]

**Comment by cvlc** [comment]
Oops, just noticed a small issue! `c3RvcmFnZW9z` is the base64-encoded string `storageos` but it won't be unencoded by the operator, so we should pass that value in as plaintext!

**Comment by Arau** [comment]
Changed base to [release/v2.6.0](https://github.com/ondat/documentation/tree/release/v2.6.0) branch rather than merge.

File: docs/reference/kubectl-plugin.md [hunk 4]
```diff
@@ -98,8 +97,21 @@ Resource declaratively, as a yaml.
...
+* Create a YAML to describe the cluster's resources using a [Helm chart](https://github.com/storageos/charts/pull/129) or use the `kubectl plugin` with the `dry-run` flags enabled: 
+
+```bash
+kubectl storageos install
+    --dry-run
+    --username storageos
+    --password storageos
+    --include-etcd ... 
+```
+  > Note, that when `--dry-run` is set for an install command, no installation takes place. Instead, the installation manifests that would have been installed under normal > circumstances are written locally to `./storageos-dry-run/`.
```
Comment (Arau) [inline docs/reference/kubectl-plugin.md:115]: there is a > in the middle of the line

**Review by cvlc — APPROVED** [review]

**Pull request merged**